src/sqlfluff/rules/L016.py

```diff
@@ -391,7 +391,6 @@ def _eval(self, segment, raw_stack, **kwargs):
 
             # We'll need the indent, so let's get it for fixing.
             line_indent = []
-            idx = 0
```

Comment:
My IDE identified this as an unused variable, so I removed it. This is not related to the fix.